BrightID/src/reducer/appsSlice.ts

```diff
@@ -14,9 +14,14 @@ const linkedContextsAdapter = createEntityAdapter<ContextInfo>({
   selectId: (linkedContext) => linkedContext.contextId,
 });
 
+const sigsAdapter = createEntityAdapter<SigInfo>({
```

Comment:
The type `SigInfo` is not declared anywhere

Reply:
sorry, I forgot that.
It can be defined based on [this](https://github.com/BrightID/BrightID/blob/13551b1ecb58fa9c6d3d2f17c706b73b79ec80de/BrightID/src/actions/apps.ts#L109-L117).

Reply:
Added type.